Entries/copy-X/copy-X.md

```diff
@@ -97,24 +97,41 @@
 Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
 
 [^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](http://web.archive.org/web/20110126034444/http://www.optimal-online.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
 
 ***
 
-## Timeline: 
+## Timeline:
+ 
+#### 1990s
+
+* 1999-01: A re-release of presumably 1998 game Meister Zufall und die Herrscher der Elemente is mastered with ZDAT dummy files, but no ring. Based on the filedates, it can be assumed these may have been present in the original release.[^47561][^Zufall_noring_txt]
+
+* 1999-05: Another re-release of presumably 1998 game Meister Zufall und die Herrscher der Elemente is mastered with ZDAT dummy files which are seemingly identical ZDAT dummy files to the previous release and presumably the 1998 release as well, and still no ring.[^47560]
 
 #### 2000s
 
 * 2000-03-02: First known reference to copy-X.[^CopyX_First_Reference].
 
+* 2000-09: Another re-release of presumably 1998 game Meister Zufall und die Herrscher der Elemente is mastered with ZDAT dummy files which are seemingly identical ZDAT dummy files to the previous release and presumably the 1998 release as well, but this time with a ring.[^Zufall_ring_txt]
+
+* 2004-02-23: First archive of Optimal Media's website advertising ProtectCD[^ProtectCD_First_Reference], already debatably advertising it more than copy-X. Optimal Media seems to have had potentially some connection[^ProtectDisc_optimal_vista][^ProtectDVD_optimal] to this DRM, and this shift is relevant because many known copy-X customers that survived to around (arbitrary date chosen) ~2008 seem to have switched[^Magix_PCD][^68141][^TZ_WWPD][^25117] to ProtectCD/ProtectDiSC.
+
 * 2009-07: As of this date, some sort of overhaul[^CopyX_2009_Document] of copy-X occured.
 
 * 2011-01-26: Last known official reference to copy-X.[^CopyX_2009_Document] (Document is from 2009, but it was hosted on their website until at least 2011)
 
+[^Magix_PCD]: [Forum post for Magix Music Maker 2008 showing error message associated with ProtectCD](https://web.archive.org/web/20241220091840/https://www.paules-pc-forum.de/forum/thread/125937-this-version-of-virtualpc-not-supported/?postID=773644)
+[^ProtectDisc_optimal_vista]: [English press article discussiong optimal media bringing ProtectDisc for its customers on Windows Vista](https://web.archive.org/web/20071022002237/http://www.optimal-online.de/fileadmin/user_upload/PDF/Presesmitteilung_Copy_Protection_windows_vista.pdf)
+[^ProtectDVD_optimal]: [German press article where optimal media claims to have outright developed ProtectDVD(-video?)]
```

Comment:
There's no link attached to this citation.

Reply:
Added missing link with https://github.com/TheRogueArchivist/DRML/pull/6/commits/f61f0d67988557ba2083b6db5ae145d8cb51c761